crates/orchestrator/src/lib.rs

```diff
@@ -30,6 +30,13 @@ use libp2p::identity::{
     ed25519::{Keypair as EdKeypair, SecretKey},
     Keypair,
 };
+
+/// Orchestrator is not, strictly speaking, bound to the network; it can have its own versioning.
+/// Orchestrator Version (major)
+pub const ORCHESTRATOR_MAJOR: u16 = 0;
```

Comment:
nit/minor (feel free to ignore): I think this should be `ORCHESTRATOR_MAJOR_VERSION`